CONTRIBUTORS.rst

```diff
@@ -171,6 +171,7 @@ is partially historical, and now, mostly arbitrary.
 - Alessandro Luongo
 - Huston Hedinger
 - Oleguer Sagarra
+- Gaetano Pietro Paolo Carpinato, Github `<https://github.com/Carghaez>`_, LinkedIn `<https://linkedin.com/in/gaetanocarpinato/>`_
```

Comment:
@Carghaez thanks for your contributions! I think the others don't put in links though, so can we reduce the listing to just your name please?

cc: @dschult, who leads maintenance and might have a different opinion from me.

Reply:
I don't think the list is too long. Let's go ahead and include links.

Reply:
Thanks for the guidance, @dschult!